### PR TITLE
added TransisObject#proxy

### PR DESCRIPTION
**Pull request by congwenma**

### CODE REVIEW

File: src/object.js [hunk 1]
```diff
@@ -482,6 +482,17 @@ TransisObject.prototype._registerProxy = function(object, name) {
   return this;
 };
 
+// Public: Exposes the internal `_registerProxy` method
```
Comment (burrows) [inline src/object.js]: We'll need better docs than this.

**Comment by burrows** [comment]
@congwenm We'll need a corresponding `Transis.Object#unproxy` method to remove a proxy. I'd also like to update `Transis.Array#proxy` to allow setting up multiple proxies on a single array. Right now that method removes the previous proxy if one exists before it sets up the new one.

File: src/object.js (hunk 1 above)
Comment (congwenm) [inline src/object.js]: lol

**Comment by burrows** [comment]
This was fixed in #51.

**Pull request closed without merging**